kustomize/move2kube/README.md

```diff
@@ -0,0 +1,76 @@
+## Prerequisites
+* The manifests are generated without the `namespace` configuration
+* Manifest names reflect the `resources` section of [kustomization.yaml](./base/kustomization.yaml)
+
+A brief explanation of the system architecture is provided in the related [document](./move2kube.md).
+
+## Kustomization options
+### Image update
+Run the following to update the default images to a custom configuration
+```
+cd overlays/prod && kustomize edit set image serverless-workflow-move2kube=quay.io/orchestrator/serverless-workflow-move2kube@1234
+cd overlays/prod && kustomize edit set image serverless-workflow-m2k-kfunc=quay.io/orchestrator/serverless-workflow-m2k-func@1234
+```
+
+## Configure properties
+Edit configuration in [replace-config](./overlays/prod/replace-config.yaml), [config.properties](./overlays/prod/config.properties) and 
+[secret.properties](./overlays/prod/secret.properties) to match your environment configuration.
+
+## Deploy to the cluster
+### Deploy the production environment
+This environment applies the generated manifests with minimal customizations to:
+* Use the latest application images
+* Force the `prod` profile for the SonataFlow instance
+* Deploy by default on the `sonataflow-infra` namespace
+* Mount the configurations defined in [Configure properties](#configure-properties) as environment variables
+
+The default namespace can be customized with:
+```bash
+TARGET_NS=YOUR-NS
+cd base && kustomize edit set namespace=$TARGET_NS && cd ..
+```
+
+Once the configuration is set, apply the deployment to the configured namespace with:
+```bash
+kustomize build overlays/prod | oc apply -f -
+```
+
+### Status check
+**TODO** Find a way to replicate the `helm status` output and chek the deployment status
+Run the following commands to wait until the move2kube workflow dependencies and build are done and workflow is running on the selected namespace:
+```
+  oc wait -n {{ .Values.namespace }}  --for=condition=Ready=true {{ $timeout }} pod -l app={{ .Values.instance.name }}
+  oc wait -n {{ .Values.namespace }} ksvc {{ .Values.kfunction.name }} --for=condition=Ready=true {{ $timeout }}
+  oc wait -n {{ .Values.namespace }} sonataflow/{{ .Values.workflow.name }} --for=condition=Running {{ $timeout }}
+```
+
+### Notes about transformation from Helm chart
+* The Helm values is converted to a `move2kube-values` ConfigMap and we use a replacement configuration to update the affected fields one by one, as in
+  [replace-config.yaml](./overlays/prod/replace-config.yaml)
+  * The original [YAML](../../charts/workflows/charts/move2kube/values.yaml) is converted to a [properties file](./overlays/prod/values.properties) aggregating the structured fields using an underscore character
+  * The original Helm variable references were transformed accordingly, e.g. `{{ .Values.instance.name }}` is changed to `instance_name`
```

Comment:
is it possible to use a more obvious and clear placeholder, like `__instance_name__` or is that a kustomize bounded thing?